virt/vm_networking/virt-networking-overview.adoc

```diff
@@ -79,7 +83,12 @@ The Data Plane Development Kit (DPDK) provides a set of libraries and drivers fo
 
 xref:../../virt/vm_networking/virt-dedicated-network-live-migration.adoc#virt-dedicated-network-live-migration[Configuring a dedicated network for live migration]::
 
+ifndef::openshift-rosa,openshift-dedicated[]
 You can configure a dedicated xref:../../virt/vm_networking/virt-connecting-vm-to-linux-bridge.adoc#virt-connecting-vm-to-linux-bridge[Multus network] for live migration. A dedicated network minimizes the effects of network saturation on tenant workloads during live migration.
+endif::openshift-rosa,openshift-dedicated[]
+ifdef::openshift-rosa,openshift-dedicated[]
+You can configure a dedicated xref:../../virt/vm_networking/virt-connecting-vm-to-linux-bridge.adoc#virt-connecting-vm-to-linux-bridge[secondary network] for live migration. A dedicated network minimizes the effects of network saturation on tenant workloads during live migration.
+endif::openshift-rosa,openshift-dedicated[]
```

Comment:
When trying to double check these conditionals, I realized that the virt-networking-overview assembly is hidden in ROSA and dedicated. Is that intentional?

Reply:
Oh! This is a new assembly. It must have been added to the OCP topic map in a rebase. I will add to ROSA/OCP.